test/e2e/storage/testsuites/provisioning.go

```diff
@@ -396,16 +469,17 @@ func TestBindingWaitForFirstConsumerMultiPVC(t StorageClassTest, client clientse
 }
 
 // runInPodWithVolume runs a command in a pod with given claim mounted to /mnt directory.
-func runInPodWithVolume(c clientset.Interface, ns, claimName, nodeName, command string, nodeSelector map[string]string, unschedulable bool) {
+func runInPodWithVolume(c clientset.Interface, ns, claimName, suffix, nodeName, command string, nodeSelector map[string]string, unschedulable bool) {
 	pod := &v1.Pod{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "Pod",
 			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: "pvc-volume-tester-",
```

Comment:
I think we should keep using GenerateName, like 

GenerateName: "pvc-volume-tester-" + suffix,

Will make it easier to find something in a log

Reply:
Because of the random number at the end? Okay, I see how that may help. I'll add it back.

Reply:
Done.